cmd/virt-launcher-monitor/virt-launcher-monitor.go

```diff
@@ -43,6 +44,8 @@ const (
 	envoyMergedPrometheusTelemetryPort = 15020
 	envoyHealthCheckPort               = 15021
 	httpRequestTimeout                 = 2 * time.Second
+
+	PasstLogFile = "/var/run/kubevirt/passt.log" // #nosec G101
```

Comment:
Do you need this one exposed? I could not see who else is using it.

Based on your findings, I would comment here on why you have not imported it from domainspec.
If not in the code, at least in the commit message.